.agent-skill-chain/adapters/claude.sh

```
@@ -13,6 +13,23 @@ ADAPTER_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
 SCRIPTS_DIR="$ADAPTER_DIR/../scripts"
 REPO_ROOT="$(cd -- "$ADAPTER_DIR/../.." &>/dev/null && pwd)"
 
+# >>> agent-skill-chain CLI resolver preamble >>>
+_ASC_WRAPPER_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+_ASC_CLI_RESOLVE_PATH="$_ASC_WRAPPER_DIR/../scripts/cli-resolve.sh"
+if [[ ! -r "$_ASC_CLI_RESOLVE_PATH" ]]; then
+  echo "agent-skill-chain CLI の共有実装を解決できません（探索パス: ${_ASC_CLI_RESOLVE_PATH}）。" >&2
+  return 1
+fi
+if ! source "$_ASC_CLI_RESOLVE_PATH"; then
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)